### ISSUE TITLE
2FA Reset doesn't work. Support not responding.

### ISSUE THREAD

**Issue by eastsida** [blockchain/blockchain-wallet-v4-frontend#3647]
I don't know where else to ask. Maybe someone will help here.
https://i.imgur.com/tuW0LA3.png
It says less than 1 minute and this message always the same even after week, two and so on and 2fa will never be reset. 2 tickets to support 2 week no response at all.

**Comment by plondon** [comment]
Did you receive an email after your request to remove 2fa?

**Comment by eastsida** [comment]
> Did you receive an email after your request to remove 2fa?

Yes, all emails on old email and new email were confirmed. In a fact my firend recently had the same problem and asked me if i have the same problem. So it's on your side 100%. No one will reset their 2fa. Until you will fix it.
Actually i can see that someone from your team can't live without bugs. I wrote already maybe about 4-5 bugs in 2 months.
1. Emails about 2fa reset didn't arrive (I think i spoke with you if Donny from support it's you) you fixed this.
2. Button "Continue" stopped working, someone just forgot to add js handler (so button was like decoration doing nothing lol) you fixed.
3. Someone forgot to put href tag in a letter and instead of buttons that redirects you to approve or deny there was unclicable text. Also fixed.
4. Now tihs one "approved less than 1 minute" hangs forever.
Do you even testing what you writing ? Do you have Q&A team ? 20mil+ customers and such bugs. Plus very nice support that replies 1-2 standard letters with auto-generated replies and when those replies doesn't help they just stop answering.
Oh look they answered on my ticket from 9th septhember about this issue, 2 weeks later.

**Comment by eastsida** [comment]
> Did you receive an email after your request to remove 2fa?

So, are you intended to do something ? 2fa reset doesn't work i can't get access to my funds. Support not responding for a month.
You doing this on purpose to steal funds ? Need to record a video where server replies "pending approval < 1 minute" and the same almost a month. All emails were confirmed. Wallet doesn't work and not even interested to fix this

**Comment by abc1abc2abc3** [comment]
Yes please record a video and bring this to large group of people to realize how blockchain is making money by firstly promoting their website which is OK but creating lots of trouble afterwards which make it difficult to withdraw or charges lots of fees or some error or letting it to hack.As blockchain security not good or probably they themselves hack our wallets.

**Comment by eastsida** [comment]
> Yes please record a video and bring this to large group of people to realize how blockchain is making money by firstly promoting their website which is OK but creating lots of trouble afterwards which make it difficult to withdraw or charges lots of fees or some error or letting it to hack.As blockchain security not good or probably they themselves hack our wallets.

No wonder why the have 2 stars no trustpilot and 2.5 on google play. The most awful support just stopped responding right after report.

**Comment by abc1abc2abc3** [comment]
Also no wonder why they hire so many people for live support to earn more by compromising our wallets.I used their live chat when I had swapping issue.That time they themselves told me to create another wallet.And when I created another wallets including exchange wallet.I started facing login issue of my private wallet.Meanwhile my identity also got rejected with no reason after using same wallet for about one year.And through live support they again misguided me to create another wallet with other email so that they can transfer my exchange fund atleast.And after creating another wallet, they started saying that they won't transfer as I have less amount.If this was then why they told this later on just to waste my time.

Also they collect our detail each time we chat.Probably they use them afterwards to access our amount.Otherwise, why would they ask any details if they don't interfare with private wallets.They should just say in beginning that they won't help

Now saying I forget my password but reality I have saved my password in notepad and Google password manager and can't be wrong.

**Comment by abc1abc2abc3** [comment]
Hi can you please delete my above comment
